### PR TITLE
216 Request for guidance on pull request protocols. I wonder why this change does not yet show on my local

### PR DESCRIPTION
**Pull request by Jelout**
## Summary by CodeRabbit

- **Documentation**
	- Updated labels for social media links in the coach details to include more descriptive text.

### CODE REVIEW

File: resources/views/livewire/coach-detail.blade.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
                 <label class="col-sm-4 col-form-label">Social media</label>
                 <div class="col-sm-8">
                     <p class="form-control-plaintext">
-                        Facebook: <a href="{{ $coach->facebook_url }}" target="_blank">{{ $coach->facebook_url }}</a>
+                        Your Social Media link: Facebook, Instagram, etc.: <a href="{{ $coach->facebook_url }}" target="_blank">{{ $coach->facebook_url }}</a>
```
Comment (jefcorral) [inline resources/views/livewire/coach-detail.blade.php:44]: This line is only for facebook link

**Review by jefcorral — CHANGES_REQUESTED** [review]
Need to refactor

**Comment by sirking1991** [comment]
I don't understand what this PR requires.
@Jelout - pls provide more context on this.

**Comment by Jelout** [comment]
> I don't understand what this PR requires. @Jelout - pls provide more context on this.

It's resolved now by Jeff. I was tying to reduce screen space

**Pull request closed without merging**